src/ex3/index.js

```diff
@@ -0,0 +1,112 @@
+import { Command } from 'commander';
+import Inquirer from 'inquirer';
+import * as ItemManager from './ItemManager.js';
+
+const program = new Command();
+
+function addTasksCli(args,options) {
+  const tasks = args.split(',');
+  if (options.pokemons) {
+     ItemManager.addPokemons(tasks);
+  }else{
+    addTasks(tasks);
+  }
+}
+
+function addTasks(tasks){
+  tasks.forEach((task) => {
+    if (!isNaN(task)) {
+      ItemManager.addPokemon(task);
+    } else {
+      ItemManager.addTodo(task);
+    }
+  });
+}
+
+function removeTask(args) {
+  if (args === '*') {
+    ItemManager.removeAll();
```

Comment:
it looks like there's a bug in the remove all? it remove only one pokemon at a time
<img width="687" alt="image" src="https://user-images.githubusercontent.com/9549288/173355437-615abd21-a810-4e83-b62b-4d51ce72c69d.png">

Reply:
I tried it again and it worked. I'm not sure why it didn't work for you.